src/config/urls.js

```diff
@@ -142,6 +142,8 @@ export const urls = {
       "https://support.ledger.com/hc/en-us/articles/360025321733?utm_source=ledger_live_desktop&utm_medium=self_referral&utm_content=error_wrongdevice",
     SyncError:
       "https://support.ledger.com/hc/en-us/articles/360012109220?utm_source=ledger_live_desktop&utm_medium=self_referral&utm_content=error_syncerror",
+    StratisDown2021Warning:
+      "https://support.ledger.com/hc/en-us/articles/115005175329-Stratis-STRAT-",
```

Comment:
zendesk uses Canonical links, no need for the title at the end. You can use:
https://support.ledger.com/hc/en-us/articles/115005175329